### ISSUE TITLE
Disregard time in train-test split

### ISSUE THREAD

**Issue by romech** [Currie32/Predicting-the-Dow-Jones-with-Headlines#6]
When making predictions on time series, train data part should precede test part. Otherwise, we can't be sure that model will work in the future.
So instead of `sklearn.model_selection.train_test_split` (which performs a random shuffle) something like `sklearn.model_selection.TimeSeriesSplit` should be used.
